orttraining/orttraining/test/training_api/core/training_api_tests.cc

```diff
@@ -104,11 +111,16 @@ TEST(TrainingApiTest, OptimStep) {
   ORT_ENFORCE(LoadCheckpoint(checkpoint_to_load_path, state).IsOK());
 
   onnxruntime::SessionOptions session_option;
+  session_option.optimized_model_filepath = "/home/bmeswani/github/onnxruntime/build/adamw_final.onnx";
```

Comment:
remove hardcoded absolute path

Reply:
Oh thanks for catching this. This was not international.